src/juicy-popover.html

```diff
@@ -128,7 +128,7 @@
                 }
             }.bind(this));
 
-            expandable.addEventListener("mouseup", function (ev) {
```

Comment:
Sorry, you are right. I haven't noticed that the listener at line 121 is for the `handle`.